src/components/User.js

```diff
@@ -0,0 +1,69 @@
+import styled from 'styled-components/macro'
+import PropTypes from 'prop-types'
+
+const data = require('../data/data.json')
+
+const users = data
```

Comment:
I would directly call the data users like 
`const data = require('../data/data.json')`